packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.tsx

```diff
@@ -228,7 +228,7 @@ const ProofOfAddressForm = observer(
 
                 const { needs_verification } = get_account_status.authentication;
                 const needs_poi = Boolean(needs_verification.length) && needs_verification.includes('identity');
-                onSubmit?.(needs_poi);
+                onSubmit?.(false);
```

Comment:
Do we really want to pass this hard coded `false` ? 👀

Reply:
Yes. Because here, I'm not passing needs_poi status as parameter, but passing whether it was a duplicate document uploaded